packages/cloog.rb

```diff
@@ -1,19 +1,40 @@
 require 'package'
 
 class Cloog < Package
-  description 'Chunky Loop Generator which is used to perform optimization in gcc'
-  homepage 'https://www.cloog.org/'
-  version '0.18.4-2'
+  description 'The CLooG Code Generator in the Polytope Model'
+  homepage 'https://github.com/periscop/cloog'
+  version '0.20.0'
+  source_url 'https://github.com/periscop/cloog/releases/download/cloog-0.20.0/cloog-0.20.0.tar.gz'
+  source_sha256 '835c49951ff57be71dcceb6234d19d2cc22a3a5df84aea0a9d9760d92166fc72'
+
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew-cross/cloog-0.18.4-2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew-cross/cloog-0.18.4-2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew-cross/cloog-0.18.4-2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew-cross/cloog-0.18.4-2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'f79bede55ba092c133a26b03c79b71a4d9e7f46c7118308f9d182f3a2ed3f2c0',
-     armv7l: 'f79bede55ba092c133a26b03c79b71a4d9e7f46c7118308f9d182f3a2ed3f2c0',
-       i686: '7bf5c2b4eb9b0d27fe10c4da4315ffb767f22dcc0281803e1f38bfbf6cbb6c74',
-     x86_64: 'e5f20db359ef15b7881f0b15f25851a462fc06ccd5e17f23cfc420a6b29b79f6',
   })
+
+  depends_on 'isl'
+  depends_on 'osl'
+
+  def self.build
+    system 'crew', 'remove', 'texinfo'
+    system 'rm', '-rf', 'isl', 'osl'
+    system 'mkdir', '-p', 'isl', 'osl'
+    system 'rm', '-f', 'autoconf/ltmain.sh', 'm4/ltversion.m4'
```

Comment:
What is wrong with just leaving this stuff alone and compile normally?

Reply:
The autoconf files give errors when running `autoreconf -i`, and we need to make sure the system libraries are used instead of the builtin ones.

Reply:
The reason it runs `autoreconf -i` is because the configure script source has been changed since they last built the configure script.